test/controllers/APIControllerSpec.scala

```diff
@@ -0,0 +1,138 @@
+package controllers
+
+import DAO.BakeryDB
+import org.scalatestplus.play._
+import org.scalatestplus.play.guice.GuiceOneAppPerTest
+import play.api.test.Helpers._
+import play.api.test.{FakeRequest, Injecting}
+import play.api.mvc._
+import java.time.LocalDate
+import java.util.UUID
+import scala.concurrent.Future
+import scala.language.postfixOps
+
+class APIControllerSpec(bakeryDB: BakeryDB)
+    extends PlaySpec
+    with Results
+    with GuiceOneAppPerTest
+    with Injecting {
+
+  "Create product" should {
+    "Return successful" in {
```

Comment:
When you make test suites they need to be descriptive. this reads
```
Create Product should
    Return Successful
    Return Unsuccessful
// etc
```

this is not an acceptable test description. "return success" _when_? returns unsuccessful _on what conditions_? 

It should return successful on a well-formed request
It should return unsuccessful on an empty object / garbage value/ partial request